web-application/random-quote-generator/styles.css

```diff
@@ -0,0 +1,45 @@
+body {
+    font-family: Arial, sans-serif;
+    background-color: #102f00;
+    margin: 0;
+    padding: 0;
+}
+
+.container {
+    max-width: 600px;
+    margin: 50px auto;
+    background-color: #ec0b0b;

```

Comment:
Try to use web safe colors and make the web page more presentable.
https://html.com/blog/design/html-colors/

Reply:
Changed the color combination.

Reply:
![Screenshot 2024-02-05 004317](https://github.com/gkmahendran09/training/assets/154506917/33f79206-9215-432c-961d-80568a5c87fd)